src/mad_spark_alt/core/multi_perspective_orchestrator.py

```
@@ -0,0 +1,530 @@
+            return HypothesisScore(0.5, 0.5, 0.5, 0.5, 0.5, 0.5)
+        
+        section = match.group(0)
+        
+        # Extract individual scores - flexible matching
+        def extract_score(patterns: List[str], text: str) -> float:
+            for pattern in patterns:
+                match = re.search(pattern, text, re.IGNORECASE)
+                if match:
+                    try:
+                        return float(match.group(1))
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)